### PR TITLE
Remove the restartdns functionality and promote the reloaddns functions

### PR DESCRIPTION
**Pull request by PromoFaux**
### **What does this PR aim to accomplish?:**

Offered as an alternative to https://github.com/pi-hole/pi-hole/pull/5779

This PR adds an additional `reloaddns-lists` to mirror the existing shortcut for `reloaddns` -> `restartDNS reload`,renames the `restartDNS` function to `reloadDNS` and removes it's ability to restart the resolver

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code and I have tested my changes.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

---
- [x] I have read the above and my PR is ready for review. _Check this box to confirm_

### CODE REVIEW

File: advanced/bash-completion/pihole [hunk 1]
```diff
@@ -7,7 +7,7 @@ _pihole() {
 
     case "${prev}" in
         "pihole")
-            opts="allow allow-regex allow-wild deny checkout debug disable enable flush help logging query reconfigure regex restartdns status tail uninstall updateGravity updatePihole version wildcard arpflush api"
+            opts="allow allow-regex allow-wild deny checkout debug disable enable flush help logging query reconfigure regex reloaddns reloaddns-lists status tail uninstall updateGravity updatePihole version wildcard arpflush api"
```
Comment (rdwebdesign) [inline advanced/bash-completion/pihole:10]: If the command `reloaddns-lists` only reload lists and keep the cache, I suggest a different name - `reloadlists`:

```suggestion
            opts="allow allow-regex allow-wild deny checkout debug disable enable flush help logging query reconfigure regex reloaddns reloadlists status tail uninstall updateGravity updatePihole version wildcard arpflush api"
```

File: manpages/pihole.8 [hunk 2]
```diff
@@ -236,16 +236,14 @@ Available commands and options:
...
 
-\fBrestartdns\fR [options]
+\fBreloaddns\fR
 .br
-    Full restart Pi-hole subsystems. Without any options (see below) a full restart causes config file parsing and history re-reading
+    Update the lists and flush the cache without restarting the DNS server
 .br
 
-    (restart options):
+\fBreloaddns-lists\fR
```
Comment (rdwebdesign) [inline manpages/pihole.8:244]: Same here:

```suggestion
\fBreloadlists\fR
```

File: pihole [hunk 3]
```diff
@@ -509,9 +504,8 @@ Options:
   enable              Enable Pi-hole subsystems
   disable             Disable Pi-hole subsystems
                         Add '-h' for more info on disable usage
-  restartdns          Full restart Pi-hole subsystems
-                        Add 'reload' to update the lists and flush the cache without restarting the DNS server
-                        Add 'reload-lists' to only update the lists WITHOUT flushing the cache or restarting the DNS server
+  reloaddns           Update the lists and flush the cache without restarting the DNS server
+  reloaddns-lists     Update the lists WITHOUT flushing the cache or restarting the DNS server
```
Comment (rdwebdesign) [inline pihole:508]: Same: 

```suggestion
  reloadlists     Update the lists WITHOUT flushing the cache or restarting the DNS server
```

File: pihole [hunk 4]
```diff
@@ -547,9 +541,9 @@ case "${1}" in
   "enable"                      ) need_root=0;;
   "disable"                     ) need_root=0;;
   "-d" | "debug"                ) ;;
-  "restartdns"                  ) ;;
   "-g" | "updateGravity"        ) ;;
   "reloaddns"                   ) ;;
+  "reloaddns-lists"             ) ;;
```
Comment (rdwebdesign) [inline pihole:546]:
```suggestion
  "reloadlists"             ) ;;
```

File: pihole [hunk 5]
```diff
@@ -590,8 +584,8 @@ case "${1}" in
   "uninstall"                   ) uninstallFunc;;
   "enable"                      ) piholeEnable true "$2";;
   "disable"                     ) piholeEnable false "$2";;
-  "restartdns"                  ) restartDNS "$2";;
-  "reloaddns"                   ) restartDNS "reload";;
+  "reloaddns"                   ) reloadDNS "reload";;
+  "reloaddns-lists"             ) reloadDNS "reload-lists";;
```
Comment (rdwebdesign) [inline pihole:588]:
```suggestion
  "reloadlists"             ) reloadDNS "reload-lists";;
```

File: pihole [hunk 6]
```diff
@@ -147,7 +147,7 @@ versionFunc() {
   exec "${PI_HOLE_SCRIPT_DIR}"/version.sh
 }
 
-restartDNS() {
+reloadDNS() {
   local svcOption svc str output status pid icon FTL_PID_FILE
   svcOption="${1:-restart}"
```
Comment (yubiuser) [inline pihole:152]: You need to change the option handling here. `restart is gone`
```suggestion
  svcOption="${1:-reload}"
```

Comment (PromoFaux) [reply pihole:152]: Good catch

**Review by yubiuser — DISMISSED** [review]

File: pihole [hunk 7]
```diff
@@ -509,9 +504,8 @@ Options:
   enable              Enable Pi-hole subsystems
   disable             Disable Pi-hole subsystems
                         Add '-h' for more info on disable usage
-  restartdns          Full restart Pi-hole subsystems
-                        Add 'reload' to update the lists and flush the cache without restarting the DNS server
-                        Add 'reload-lists' to only update the lists WITHOUT flushing the cache or restarting the DNS server
+  reloaddns           Update the lists and flush the cache without restarting the DNS server
+  reloadlists     Update the lists WITHOUT flushing the cache or restarting the DNS server
```
Comment (rdwebdesign) [inline pihole:508]: Just using the same indentation for every item:

```suggestion
  reloadlists         Update the lists WITHOUT flushing the cache or restarting the DNS server
```

File: pihole [hunk 8]
```diff
@@ -547,9 +541,9 @@ case "${1}" in
   "enable"                      ) need_root=0;;
   "disable"                     ) need_root=0;;
   "-d" | "debug"                ) ;;
-  "restartdns"                  ) ;;
   "-g" | "updateGravity"        ) ;;
   "reloaddns"                   ) ;;
+  "reloadlists"             ) ;;
```
Comment (rdwebdesign) [inline pihole:546]: Same:

```suggestion
  "reloadlists"                 ) ;;
```

File: pihole [hunk 9]
```diff
@@ -590,8 +584,8 @@ case "${1}" in
   "uninstall"                   ) uninstallFunc;;
   "enable"                      ) piholeEnable true "$2";;
   "disable"                     ) piholeEnable false "$2";;
-  "restartdns"                  ) restartDNS "$2";;
-  "reloaddns"                   ) restartDNS "reload";;
+  "reloaddns"                   ) reloadDNS "reload";;
+  "reloadlists"             ) reloadDNS "reload-lists";;
```
Comment (rdwebdesign) [inline pihole:588]: Same:

```suggestion
  "reloadlists"                 ) reloadDNS "reload-lists";;
```

**Review by rdwebdesign — CHANGES_REQUESTED** [review]

File: pihole (hunk 7 above)
Comment (PromoFaux) [reply pihole:508]: I've aligned them all in one fell swoop!

**Review by rdwebdesign — APPROVED** [review]

**Pull request merged**